app/indexers/default_work_indexer.rb

```diff
@@ -16,7 +16,12 @@ def generate_solr_document
      rights_statement_labels = ScholarsArchive::RightsStatementService.new.all_labels(object.rights_statement)
      license_labels = ScholarsArchive::LicenseService.new.all_labels(object.license)
      language_labels = ScholarsArchive::LanguageService.new.all_labels(object.language)
-
+     triple_powered_fields = [:academic_affiliation, :other_affiliation]
```

Comment:
you should be able to eliminate this and the `.each` to use the object

`object.triple_powered_properties.each.....`